independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -1,9 +1,21 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    {#if parent}
+    <parent>
+        <groupId>{project.parent.group-id}</groupId>
+        <artifactId>{project.parent.artifact-id}</artifactId>
+        <version>{project.parent.version}</version>
+    </parent>
+    {/if}
+
+    {#if project.group-id not blank}
     <groupId>{project.group-id}</groupId>
+    {#if parent}
     <artifactId>{project.artifact-id}</artifactId>
+    {#if project.version not blank}
     <version>{project.version}</version>
+    {#if parent}
```

Comment:
I think there is a problem with the closing `{/if}`

Reply:
Sure, I wasn't expecting this to work, but it should be something like that I imagine. If you check the comments I made in the issue I guess it would be clearer